crates/glaredb/src/local.rs

```diff
@@ -73,25 +73,18 @@ impl LocalSession {
                     format!("Connected to remote GlareDB server: {}", u.cyan()),
                 )
             } else {
-                let tls_conf = if opts.disable_tls {
-                    None
-                } else {
-                    match (opts.ca_cert_path.clone(), opts.domain.clone()) {
-                        (Some(ca_cert_path), Some(domain)) => Some(TlsConfig {
-                            ca_cert_path,
-                            domain,
-                        }),
-                        _ => {
-                            return Err(anyhow!(
-                                "Specify ca-cert-path and domain in --args for TLS"
-                            ));
-                        }
-                    }
-                };
+                let mut dst: ProxyDestination = url.try_into()?;
+
+                // Review: I had wanted to do this in
+                // RemoteClient::connect_with_proxy_auth_params
+                if opts.enable_tls {
+                    dst.dst
+                        .set_scheme("https")
+                        .expect("failed to upgrade scheme from http to https");
+                }
 
```

Comment:
> this URL is the cloud api for the proxy, or is this the API URL that we're getting the cert from?

I feel confused by the former, so I'll try to answer:

This is for calling rpc proxy (ie: https://remote.glared.com:...), and is performed by tonic:

https://github.com/GlareDB/glaredb/blob/grey/tls/crates/sqlexec/src/remote/client.rs#L219

To see it's exact shape, see here:

https://github.com/GlareDB/glaredb/blob/8dddb8262d75cf1943b1033a3306fc2aa30834b6/crates/sqlexec/src/remote/client.rs#L416-L434

Reply:
I think part of my confusion is that I typically don't think about grpc as being HTTPS (even though it is; I think it will do http, but it downcasts to json, which you often want to avoid)

Reply:
My understanding is that we need to change the scheme from `http` -> `https` here, and the assumption is based purely on the tonic example:

https://github.com/hyperium/tonic/blob/d5c14fa20623a189771d993cb019697565393df6/examples/src/tls/client.rs#L18-L19